examples/2d/mesh2d_transforms.rs

```diff
@@ -0,0 +1,91 @@
+//! Used to test transforms with mesh2d
+//! This makes sure the depth buffer is correctly being used for opaque and transparent 2d meshes
```

Comment:
This needs an explanation somewhere in the code about what a "correct" result looks like.

Reply:
done